src/main/java/com/javadiscord/jdi/internal/models/message/MessageReactionsRemoved.java

```diff
@@ -5,7 +5,7 @@
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public record MessageReactionsRemoved(
-    @JsonProperty("message_id") long message_id,
+    @JsonProperty("messageId") long messageId,
```

Comment:
Shouldn't this be message_id

Reply:
yeah strange it got changed in this PR, it was accidental